Buddies/Onboarding/SignUpInfoVC.swift

```diff
@@ -68,6 +68,8 @@ class SignUpInfoVC: LoginBase, UIImagePickerControllerDelegate, UINavigationCont
         bioText.layer.borderColor = ControlColors.fieldBorderFocused.cgColor
         bioText.layer.borderWidth = 2
         buttonPicture.layer.cornerRadius = buttonPicture.frame.size.width / 2
+        
+        LocationPersistence.instance.makeSureWeHaveLocationAccess(from: self)
```

Comment:
another quick note here, I _think_ that this popup is going to "conflict" with the notification request. It's not a big deal - it just feels bad.